crmsh/ui_configure.py

```diff
@@ -57,25 +57,28 @@ def _tag_completions():
 _rsc_template_list = compl.call(cib_factory.rsc_template_list)
 _container_type = compl.choice(constants.container_type)
 
+
 def _advanced_completer(args):
     '''
     meta completers for group/ms/clone resource type 
     '''
     key_words = ["meta", "params"]
     completing = args[-1]
     resource_type = args[0]
+    return_list = []
     if completing.endswith('='):
         # TODO add some help messages
-        return []
+        return return_list
```

Comment:
`return_list` is always `[]` here. Why make this change? It seems confused to me.

Reply:
I've changed it back:)